mlflow/models/container/__init__.py

```diff
@@ -128,12 +127,11 @@ def _serve_pyfunc(model):
     # link the log streams to stdout/err so they will be logged to the container logs
     check_call(['ln', '-sf', '/dev/stdout', '/var/log/nginx/access.log'])
     check_call(['ln', '-sf', '/dev/stderr', '/var/log/nginx/error.log'])
-    cpu_count = multiprocessing.cpu_count()
+
     os.system("pip -V")
     os.system("python -V")
     os.system('python -c"from mlflow.version import VERSION as V; print(V)"')
-    cmd = ("gunicorn --timeout 60 -k gevent -b unix:/tmp/gunicorn.sock -w {nworkers} " +
-           "mlflow.models.container.scoring_server.wsgi:app").format(nworkers=cpu_count)
+    cmd = "gunicorn ${GUNICORN_CMD_ARGS} mlflow.models.container.scoring_server.wsgi:app"
```

Comment:
This makes sense - it is a little weird though that if a user wants to override default gunicorn args, they must include `-b `unix:/tmp/gunicorn.sock`. Would you be able to investigate if there's a way to set `-b unix:/tmp/gunicorn.sock` for the user? For example, maybe we can still set `-b unix:/tmp/gunicorn.sock` here,  pass GUNICORN_CMD_ARGS through as an environment variable when we actually launch the gunicorn process [here](https://github.com/mlflow/mlflow/pull/1428/files#diff-b5a2dc73d42c2570e28dc900fc6eaf38L138), and gunicorn will respect the command-line argument over the environment variable?

Reply:
Actually, an even cleaner solution might be to remove the `-b` option altogether from our container, which would cause the `gunicorn` server to run at 127.0.0.1:8000 by default. We could then update [this line](https://github.com/mlflow/mlflow/blob/706977f0b6b918906722909c8f3c197e5cf1712e/mlflow/models/container/scoring_server/nginx.conf#L18) of our nginx conf to `server 127.0.0.1:8080` to forward requests to our gunicorn server, as described by [nginx docs](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#upstream)

Reply:
Good point that the server is determined by nginx. Using the default localhost now. Thanks!